docs/conf.py

```diff
@@ -0,0 +1,323 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
```

Comment:
I note that this is still not fully done... If you are busy with CAP demo, we can change it after merge perhaps.